random_pick_depends.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+export DEPEND=/root/depends
```

Comment:
I would never want to run this on my machine. It tries to automate too much. It would be better to add these steps to a README and let the user decide how to setup their environment

Reply:
Ok, I see now, this is meant to be run on travis not on a user's machine. In that case, I would recommend cleaning this up a bit so that it is more readable and maintainable and renaming it `travis_before_script.sh` It may also make sense to put it in a subfolder `.travis/before_script.sh` to make it extremely clear to a user that this is not meant to be run on their local machine

Reply:
Truly, I only use this file to travis. Great idea to put it under `.travis` subfolder! Now, the file locates in `.travis/random_pick/`. I think maybe each application can provide a similar file under `.travis/<app_name>/`. In addition, I'm also thinking maybe each application can have its own docker file under `.docker/<app_name>` folder.